core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1,1423 +1,1330 @@
-# The private key here is used to mirror content from the ops mirror and the redhat CDN
-- from:
-    ops-mirror.pem:
-      bw_item: mirror.openshift.com
-      attachment: cert-key.pem
-    rh-cdn.pem:
-      bw_item: rh-cdn
-      attachment: rh-cdn.pem
-  to:
-    - cluster: api.ci
-      namespace: ocp
-      name: mirror.openshift.com
-    - cluster: build01
-      namespace: ocp
-      name: mirror.openshift.com
-    - cluster: build02
-      namespace: ocp
-      name: mirror.openshift.com
-    - cluster: vsphere
-      namespace: ocp
-      name: mirror.openshift.com
-# Host keys for the SSHD bastions
-- from:
-    authorized_keys:
-      bw_item: sshd-bastion-z
-      attachment: authorized_keys
-  to:
-    - cluster: api.ci
-      namespace: bastion-z
-      name: sshd-authorized-keys
-- from:
-    ssh_host_rsa_key:
-      bw_item: sshd-bastion-z
-      attachment: ssh_host_rsa_key
-    ssh_host_dsa_key:
-      bw_item: sshd-bastion-z
-      attachment: ssh_host_dsa_key
-    ssh_host_ecdsa_key:
-      bw_item: sshd-bastion-z
-      attachment: ssh_host_ecdsa_key
-    ssh_host_ed25519_key:
-      bw_item: sshd-bastion-z
-      attachment: ssh_host_ed25519_key
-  to:
-    - cluster: api.ci
-      namespace: bastion-z
-      name: sshd-host-keys
-- from:
-    authorized_keys:
-      bw_item: sshd-bastion-ppc64le
-      attachment: authorized_keys
-  to:
-    - cluster: api.ci
-      namespace: bastion-ppc64le
-      name: sshd-authorized-keys
-- from:
-    ssh_host_rsa_key:
-      bw_item: sshd-bastion-ppc64le
-      attachment: ssh_host_rsa_key
-    ssh_host_dsa_key:
-      bw_item: sshd-bastion-ppc64le
-      attachment: ssh_host_dsa_key
-    ssh_host_ecdsa_key:
-      bw_item: sshd-bastion-ppc64le
-      attachment: ssh_host_ecdsa_key
-    ssh_host_ed25519_key:
-      bw_item: sshd-bastion-ppc64le
-      attachment: ssh_host_ed25519_key
-  to:
-    - cluster: api.ci
-      namespace: bastion-ppc64le
-      name: sshd-host-keys
+cluster_groups:
+  all:
```

Comment:
Whats the reason for having these three groups? Logically, we only have one possible group: `build-clusters` (and they all need the same secrets, e.G. we also need to put the private repo cloner onto the `vsphere` cluster)

Also, a group with only a single member doesn't really make sense and `others` is a bad name because it doesn't describe whats in it :)

Reply:
We treat app.ci differently, right?
Like we do not plan to run real testing jobs on it.

Reply:
app.ci is a good point. It needs many but not all things that are needed for all build clusters. I personally tend to just treat it like a build cluster so we don't have to remember to also add it when adding new secrets but that is debatable

Reply:
OK. I will do `build-farm` and `non-app-ci`.